encoding/preparation.md

```diff
@@ -339,6 +339,11 @@ And they both have a point.
 
 So, do some ABX testing and decide for yourself.
 
+Be aware though,
+encoding a lossy audio-track with a lossless codec
+will result in the same lossy audio-track.
+It will require a lot more bitrate though. 
```

Comment:
This should be worded more strongly, like "However, it is never an option to transcode lossy audio into lossles because you are using more bitrate for the same information. You cannot generate the information that was lost during compression."